README.md

```diff
@@ -15,6 +15,66 @@ automation and interaction capabilities for developers and tools.
 ## Prerequisites
 
 1. To run the server in a container, you will need to have [Docker](https://www.docker.com/) installed.
+
+## Required Token Permissions
```

Comment:
Please remove this section. I think it's interesting to have, but should have its own issue for discussion.